.github/workflows/test.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         run: ./gradlew classes
       - name: Assemble
         run: ./gradlew assemble
+      - name: Validate Docker Configurations
+        run: find . -name docker-compose\*.yml | while read -r file; do docker-compose -f "$file" config -q; done
```

Comment:
Interestingly, `find . -name docker-compose\*.yml -exec docker compose -f {} config -q \;` just returned:
```
services.aerie_merlin.build.args must be a mapping
```
Instead of:
```
ERROR: The Compose file '././docker-compose.yml' is invalid because:
services.aerie_merlin.build.args contains an invalid type, it should be an object, or an array
```
When the following config. error was (intentionally) introduced:
```diff
diff --git a/docker-compose.yml b/docker-compose.yml
index 78ccc3a35..4f3ef7405 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -51,7 +51,7 @@ services:
       context: ./merlin-server
       dockerfile: Dockerfile
       args:
-        IMAGE_INCLUDE_JDK: "true"
+        IMAGE_INCLUDE_JDK "true"
     container_name: aerie_merlin
     depends_on: ["postgres"]
     environment:
```

Either way, the `find ... | while read ...` pattern is more than good enough here.

Reply:
lol – it's a `docker-compose` vs `docker compose` thing 😂 my bad.

Reply:
I really prefer the `docker-compose` output since it actually indicates which file is invalid. I'm remedy this by adding an `echo` prior to `docker compose config`.